### PR TITLE
change title on README

### PR DESCRIPTION
**Pull request by bprakashputta**
* Left reviews and line comments
* Created a file and made a commit
* Opened a pull request

### CODE REVIEW

**Review by bprakashputta — COMMENTED** [review]
Good

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
```
Comment (theharrypotter1999) [inline README.md:1]: Changed game title

Comment (bprakashputta) [inline README.md:1]: Changed title of game

**Review by bprakashputta — COMMENTED** [review]
changed title of game, updating the Readme file

**Pull request merged**